packages/jest-diff/src/index.ts

```diff
@@ -163,16 +159,4 @@ function compareObjects(
   return diffMessage;
 }
 
-// eslint-disable-next-line no-redeclare
-namespace diff {
-  export type Diff = DiffClass;
-  export type DiffOptions = JestDiffOptions;
-}
-
-diff.diffStringsUnified = diffStringsUnified;
-diff.diffStringsRaw = diffStringsRaw;
-diff.DIFF_DELETE = DIFF_DELETE;
-diff.DIFF_EQUAL = DIFF_EQUAL;
-diff.DIFF_INSERT = DIFF_INSERT;
-
-export = diff;
+export default diff;
```

Comment:
This is a breaking changes (for people using `require`). Mind adding a changelog entry for it?

Reply:
Oh yeah, for sure. I renamed pull request so people can see that is the important change.